app/templates/labs.html

```diff
@@ -84,8 +140,9 @@ <h1 class="page-title">ADI Labs</h1>
            <h4>FAQ</h4>
            <span class="bar green"></span>
        </div>
+       <h3>Check back soon!</h3>
    </div>
-   <div class="content">
+<!--    <div class="content">
```

Comment:
it'll come back once we have the FAQ done

On Mon, Sep 14, 2015 at 2:35 PM Nate <notifications@github.com> wrote:

> In app/templates/labs.html
> <https://github.com/adicu/adi-website/pull/240#discussion_r39429687>:
>
> >     </div>
> > -   <div class="content">
> > +<!--    <div class="content">
>
> is there a reason to keep this if it is in a comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adicu/adi-website/pull/240/files#r39429687>.
>